pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -4381,3 +4382,83 @@ func TestMergeDBDeployment(t *testing.T) {
 		})
 	}
 }
+
+func TestJvmCrd(t *testing.T) {
```

Comment:
take a look at these crd schema tests in this file... maybe you can instead replicate what we already do here to accomplish the same thing for jvm specific crd validation?
https://github.com/kiegroup/kie-cloud-operator/blob/d6954019fcf87b0e2c452a5fc842b22de57a4850/pkg/controller/kieapp/test/crd_validation_test.go

Reply:
Seems similar but the purpose of this test is to fail if the expected jvm params are removed from crd, if someone change the yaml has to change the test to alter the params

Reply:
Ok